
bool STOP_RESUME
bool IO_START
bool STOP_PROTECTIVE
bool STOP_EMERGENCY
bool STOP_MANUAL
bool MODE_LOCK
